client/my-sites/stats/activity-log/index.jsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Main from 'components/main';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSiteSlug, isJetpackSite } from 'state/sites/selectors';
+import StatsFirstView from '../stats-first-view';
+import SidebarNavigation from 'my-sites/sidebar-navigation';
+import StatsNavigation from '../stats-navigation';
+
+class ActivityLog extends Component {
+	componentDidMount() {
+		window.scrollTo( 0, 0 );
```

Comment:
I observed that we tend to scroll to the top early in the controllers and not in the components.

Also, if you decide to put that one in the controller, this class will become simple enough to be refactored to a stateless functional component. But that's only in case you're not planning to further expand it as activity log is being developed.